mmv1/products/accesscontextmanager/AuthorizedOrgsDesc.yaml

```diff
@@ -0,0 +1,129 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Api::Resource
+  name: 'AuthorizedOrgsDesc'
+  base_url: ""
+  create_url: "{{parent}}/authorizedOrgsDescs"
+  self_link: "{{name}}"
+  update_verb: :PATCH
+  references: !ruby/object:Api::Resource::ReferenceLinks
+    guides:
+      'gcloud docs': 'https://cloud.google.com/beyondcorp-enterprise/docs/cross-org-authorization'
```

Comment:
This link 404s.

Reply:
Yup, that's where the guide will eventually be but it's not there yet. Currently it's only available internally here: https://cloud.devsite.corp.google.com/beyondcorp-enterprise/docs/cross-org-authorization

Should I remove the link to the guide until it's available?

Reply:
The user guide is now live at the URL in the code so it no longer 404s.